core/storage/email/gae_models.py

```diff
@@ -394,12 +408,17 @@ def get_deletion_policy():
         """Feedback email reply to id should be deleted."""
         return base_models.DELETION_POLICY.DELETE
 
+    @staticmethod
+    def get_export_method():
+        """Model is exported as a single unshared instance."""
+        return base_models.EXPORT_METHOD.MULTIPLE_UNSHARED_INSTANCES
```

Comment:
Docstring contradicts return value. You might want to check the rest of the PR for cases like this...

Reply:
Done